webrtc.html

```diff
@@ -5228,18 +5225,10 @@ <h2>Methods</h2>
                   <code>true</code>, <a>throw</a> an
                   <code>InvalidStateError</code>.</p>
                 </li>
-                <li>
-                  <p>Verify that each <code><a data-link-for="RTCRtpCodingParameters">rid</a></code>
-                  value in <var>sendEncodings</var> is composed only of
-                  alphanumeric characters (a-z, A-Z, 0-9) up to
-                  a maximum of 16 characters. If one of the RIDs does not meet
-                  these requirements, <a>throw</a> a <code>TypeError</code>.</p>
-                </li>
```

Comment:
Here and elsewhere: Instead of the verify steps, should this say run the verification steps defined in the other document? Now it looks like we're just skipping them.

Reply:
Perhaps the right thing is to refer readers to the other specification in the situation where there are multiple encodings.